app/models/state_file_dependent.rb

```diff
@@ -32,4 +37,9 @@ def full_name
     parts << suffix if suffix.present?
     parts.compact.join(' ')
   end
+
+  def ask_senior_questions?
+    return false if dob.nil?
+    dob <= DateTime.new(2023, 12, 31).years_ago(65) && months_in_home == 12 && (relationship == 'PARENT' || relationship == 'GRANDPARENT')
```

Comment:
It might be helpful to pull out the end of tax year value into some global thing